app/queries/overdue_patients_query.rb

```diff
@@ -0,0 +1,24 @@
+class OverduePatientsQuery
+  def count_patients_called(region, period_type, group_by: nil)
+    query = Reports::OverduePatient
+      .group_by_period(period_type, :month_date, {format: Period.formatter(period_type)})
+      .where(has_called: "yes")
+      .where("assigned_facility_region_id in (?)", region.facility_regions.pluck(:id))
+    if group_by.present?
+      results = query.group(group_by).count
+      sum_groups_by_period(results)
+    else
+      query.count
+    end
+  end
+
+  private
+
+  def sum_groups_by_period(result)
```

Comment:
Why is this called `sum_groups_by_period`? We are not summing anything here.

Reply:
It's the `sum_groups` by period

Reply:
It's probably a bad name, but consistent with other _query files